yarn.lock

```diff
@@ -0,0 +1,6257 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
You added `yarn.lock` to the `gitignore` but you're pushing it up in this commit. Do you want to track this file or not?

Reply:
Meant to ignore it as a measure of safety as I'm not sure if it's desirable to track it or not. What's your opinion?

Reply:
You should commit the lock file. In which case we don't include it in gitignore.

Reply:
done.